src/document.rs

```diff
@@ -295,6 +298,34 @@ impl Document {
         }
     }
 
+    /// Metadata for generating JSON feeds
+    pub fn to_jsonfeed(&self, root_url: &str) -> jsonfeed::Item {
+        let description = self.attributes
+            .get("description")
+            .or_else(|| self.attributes.get("excerpt"))
+            .or_else(|| self.attributes.get("content"))
+            .and_then(|s| s.as_str())
+            .map(|s| s.to_owned());
+
+        let link = root_url.to_owned() + &self.path.replace("\\", "/");
+
+        Item {
+            id: "1".into(),
```

Comment:
Yes, I have added my TODOs to the git comment. Will move them into source in the next revision.